lib/Field/FieldLeft.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import { node, string } from 'prop-types';
+import { FieldInStructureEditModeContext } from './FieldInStructureEditModeProvider';
+
+export const FieldLeft = ({ children, className }) => {
+  const { inStructureEditMode } = useContext(FieldInStructureEditModeContext);
+
+  return (
+    <Col
+      xs={12}
+      sm={inStructureEditMode ? 3 : 4}
```

Comment:
Why do we need to change this prop for the editable state in `FieldLeft` and not have to change anything for what is currently called `FieldMiddle`?

`sm` in middle is currently fixed at `8` so this should always be `4` to make up a `12` grid.